Inclet/javadevs--reactNative
# Home

#### What does this PR do?
This feature allows a user to see all java developers and can click on one of them to view their profile.
#### Description of Task to be completed?
- Create home component.
- Create Profile component.
- Added tests for the components.
- Added apollo and graphQl.

#### How should this be manually tested?
- First, clone the repo from https://github.com/Inclet/Javadevs--reactNative.git if you haven't.

- Checkout to `home` branch by typing `git checkout home` in your terminal.

- Run `git pull` to pull all changes in the current branch.

- Open `javadevs.xcworkspace` in your Xcode from ios folder where you cloned the repo.
Then start the app by hitting the Run button in Xcode. Automatically, the welcome screen will show up your ios simulator. By clicking the forward under the welcome text on the first screen you will automatically be directed to About screen. From here, you click the blue button labeled `Let's Go` and it will direct you to auth screen. from here you can choose login button, from here, you can log in with any social account and after successful login, you will see a list of all developers.
#### Screenshots (if appropriate)
<img width="398" alt="Screen Shot 2019-08-20 at 12 10 02" src="https://user-images.githubusercontent.com/37753967/63338700-97e36c80-c343-11e9-9afb-0a47355dd3dd.png">
<img width="388" alt="Screen Shot 2019-08-20 at 12 10 38" src="https://user-images.githubusercontent.com/37753967/63338702-97e36c80-c343-11e9-9d0f-9525105af767.png">
